### ISSUE TITLE
Packer crashed on Windows googlecompute

### ISSUE THREAD

**Issue by angerson** [hashicorp/packer#8139]
- Packer version v1.4.3
- Ubuntu 18.04.3 LTS
- Crash gist: https://gist.github.com/angersson/5b4ffd221fda0480bade7922b1664f2f

Snippet:
```
2019/09/20 21:29:44 ui: [0;32m    googlecompute: Instance has been deleted![0m
2019/09/20 21:29:44 ui: [1;32m==> googlecompute: Deleting disk...[0m
2019/09/20 21:29:45 packer: 2019/09/20 21:29:45 Retryable error: retrying for state DONE, got RUNNING
2019/09/20 21:29:47 ui: [0;32m    googlecompute: Disk has been deleted![0m
2019/09/20 21:29:47 packer: panic: runtime error: invalid memory address or nil pointer dereference
2019/09/20 21:29:47 packer: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1cc79f7]
2019/09/20 21:29:47 packer: 
2019/09/20 21:29:47 packer: goroutine 32 [running]:
2019/09/20 21:29:47 packer: github.com/hashicorp/packer/builder/googlecompute.(*StepCreateWindowsPassword).Run(0xc000526120, 0x3fadba0, 0xc0005a1c40, 0x3f936a0, 0xc0004187b0, 0x0)
2019/09/20 21:29:47 packer: 	/Users/mmarsh/Projects/packer/builder/googlecompute/step_create_windows_password.go:63 +0x657
2019/09/20 21:29:47 packer: github.com/hashicorp/packer/helper/multistep.(*BasicRunner).Run(0xc000418840, 0x3fadba0, 0xc0005a1c40, 0x3f936a0, 0xc0004187b0)
2019/09/20 21:29:47 packer: 	/Users/mmarsh/Projects/packer/helper/multistep/basic_runner.go:67 +0x1ec
2019/09/20 21:29:47 packer: github.com/hashicorp/packer/builder/googlecompute.(*Builder).Run(0xc0002fc0a0, 0x3fadba0, 0xc0005a1c40, 0x3fc97a0, 0xc0002fdb60, 0x3f46f40, 0xc00059fff0, 0x40b9b9, 0xc000381d10, 0x30, ...)
2019/09/20 21:29:47 packer: 	/Users/mmarsh/Projects/packer/builder/googlecompute/builder.go:86 +0x95a
2019/09/20 21:29:47 packer: github.com/hashicorp/packer/packer/rpc.(*BuilderServer).Run(0xc0003813b0, 0x1, 0xc000492450, 0x0, 0x0)
2019/09/20 21:29:47 packer: 	/Users/mmarsh/Projects/packer/packer/rpc/builder.go:109 +0x231
2019/09/20 21:29:47 packer: reflect.Value.call(0xc000304780, 0xc0003b6368, 0x13, 0x3998dd5, 0x4, 0xc00010df18, 0x3, 0x3, 0xc0005a0300, 0x3, ...)
2019/09/20 21:29:47 packer: 	/usr/local/go/src/reflect/value.go:447 +0x461
2019/09/20 21:29:47 packer: reflect.Value.Call(0xc000304780, 0xc0003b6368, 0x13, 0xc00002a718, 0x3, 0x3, 0xc00057ed80, 0xc00002a730, 0x404e35)
2019/09/20 21:29:47 packer: 	/usr/local/go/src/reflect/value.go:308 +0xa4
2019/09/20 21:29:47 packer: net/rpc.(*service).call(0xc0002fa800, 0xc0005b80a0, 0xc0003b82d0, 0xc0003b82e0, 0xc00049c380, 0xc00059c440, 0x2ecd180, 0xc00049241c, 0x18a, 0x2e1fce0, ...)
2019/09/20 21:29:47 packer: 	/usr/local/go/src/net/rpc/server.go:384 +0x14e
2019/09/20 21:29:47 packer: created by net/rpc.(*Server).ServeCodec
2019/09/20 21:29:47 packer: 	/usr/local/go/src/net/rpc/server.go:481 +0x42b
```

(presumably mmarsh built the packer binary, or something.) 

Since packer told me to report the crash as-is (I got the `!!!!! PACKER CRASH !!!!!` message), I'm assuming I don't need to provide a minimum viable reproduction for the log to still be useful. Let me know if you still need it and I'll see if I can reproduce it.

**Comment by SwampDragons** [comment]
Thanks for reporting!  I believe this is a duplicate of https://github.com/hashicorp/packer/issues/8085, which has been resolved and will be released with 1.4.4 . I'll leave this issue open for now to catch any further people who run across the bug, and will close it when we release.

**Comment by SwampDragons** [comment]
Closed in the 1.4.4 release
